Resources/views/standard_layout.html.twig

```diff
@@ -8,18 +8,18 @@ For the full copyright and license information, please view the LICENSE
 file that was distributed with this source code.
 
 #}
-{% set _preview              = block('preview') %}
-{% set _form                 = block('form') %}
-{% set _show                 = block('show') %}
-{% set _list_table           = block('list_table') %}
-{% set _list_filters         = block('list_filters') %}
-{% set _tab_menu             = block('tab_menu') %}
-{% set _content              = block('content') %}
-{% set _title                = block('title') %}
-{% set _breadcrumb           = block('breadcrumb') %}
-{% set _actions              = block('actions') %}
-{% set _navbar_title         = block('navbar_title') %}
-{% set _list_filters_actions = block('list_filters_actions') %}
+{% set _preview              = block('preview') is defined ? block('preview') : null %}
```

Comment:
`is defined` is a strange construct that does a lot more than checking if it's `null`. In Twig 1.28, they also added `is defined` support for `block()` calls: https://github.com/twigphp/Twig/pull/2239

So instead of looking at it from a `block('preview') === null` point of view, you should read it as `block preview is defined`.

About `|default()`, it uses `twig_test_empty()`, which doesn't use the `is defined` construct but just compares blank values against the object. So it wouldn't work.